docs/Developers/Newcomers_Guide.rst

```diff
@@ -336,9 +336,16 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
 .. note::
 
-    **Do not only fix a newcomer issue!** Supervising newcomers is really a lot
-    of work. We're all volunteers and we can't keep this up if you don't help
-    us in other areas as well!
```

Comment:
I think we should keep the "this is a lot of work sentences", I think they are really important to tell the newcomers to be a bit more encouraged when helping out here, because it's indeed a lot of work :) I think it's well integratable below

> **Do not only fix a newcomer issue!** It is highly recommmended that you
> fix one newcommer issue to get familiar with the workflow at coala and
> then proceed to a difficulty/low issue. [... now the new sentences]